app/src/main/java/com/javahelp/frontend/fragments/AccountFragment.java

```diff
@@ -0,0 +1,62 @@
+package com.javahelp.frontend.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.fragment.app.Fragment;
+
+import com.javahelp.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link AccountFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class AccountFragment extends Fragment {
+
+    // This is example code. These will be changed later
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    private String mParam1;
+    private String mParam2;
+
+    public AccountFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param param1 Parameter 1.
+     * @param param2 Parameter 2.
+     * @return A new instance of fragment AccountFragment.
+     */
+    public static AccountFragment newInstance(String param1, String param2) {
+        AccountFragment fragment = new AccountFragment();
+        Bundle args = new Bundle();
+        args.putString(ARG_PARAM1, param1);
+        args.putString(ARG_PARAM2, param2);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
```

Comment:
I think it is OK not to worry about when `getArguments` from `Fragment` superclass to return `null`, because it is difficult to handle that.